doc: Minor fixes to docs (#17986)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf91/nrf91_snippet.rst

The ``nrf91-modem-trace-ext-flash`` snippet enables modem tracing, the flash backend, and external flash and configures them to store modem traces to a dedicated partition on the external flash for supported boards.
To change the partition size, the project needs to configure the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_FLASH_PARTITION_SIZE` Kconfig option.

To enable modem traces with the flash backend, add the ``nrf91-modem-trace-ext-flash`` snippet to the :term:`build configuration`.

> This can be done in one of the following ways:

Reviewer: ```suggestion
You can do this in one of the following ways:
```

The ``nrf91-modem-trace-ram`` snippet enables modem tracing and configures it to store modem traces to a dedicated partition on the RAM.
To change the partition size, the project needs to configure the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_RAM_LENGTH` Kconfig option.

To enable modem traces with the RAM backend, add the ``nrf91-modem-trace-ram`` snippet to the :term:`build configuration`.

> This can be done in one of the following ways:

Reviewer: ```suggestion
You can do this in one of the following ways:
```

The ``nrf91-modem-trace-rtt`` snippet enables the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option and chooses the RTT trace backend with the necessary Kconfig options.

To enable modem tracing with the RTT backend, add the ``nrf91-modem-trace-rtt`` snippet to the :term:`build configuration`.

> This can be done in one of the following ways:

Reviewer: ```suggestion
You can do this in one of the following ways:
```

.. tabs::

   .. group-tab:: west

      To add modem traces with the RTT backend when building an application with west, use the following command pattern, where *board_target* corresponds to your board target and `<image_name>` to your application image name:

      .. parsed-literal::
         :class: highlight

         west build --board *board_target* -- -D<image_name>_SNIPPET="nrf91-modem-trace-rtt"

   .. group-tab:: CMake

       To add the modem traces with the RTT backend snippet when building an application with CMake, add the following command to the CMake arguments:

       .. code-block:: console

          -D<image_name>_SNIPPET="nrf91-modem-trace-rtt" [...]

       To build with the |nRFVSC|, specify ``-D<image_name>_SNIPPET="nrf91-modem-trace-rtt" [...]`` in the **Extra CMake arguments** field.

       See :ref:`cmake_options` for more details.

.. _tfm_enable_share_uart:

Shared UART for application and TF-M logging
********************************************

> If you want to activate TF-M logging while having modem traces enabled, it can be useful to direct the TF-M logs to the UART (**UART0**) used by the application.

Reviewer 2: ```suggestion
If you want to activate TF-M logging while having modem traces with the UART backend enabled, it can be useful to direct the TF-M logs to the UART (**UART0**) used by the application.
```

To enable modem traces and TF-M logs, add the ``nrf91-modem-trace-uart`` and ``tfm-enable-share-uart`` snippets to the :term:`build configuration`.

> This can be done in one of the following ways:

Reviewer: ```suggestion
You can do this in one of the following ways:
```
